src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -379,7 +379,7 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
           fields.get("status").isDefined &&
           fields.get("status").get.asInstanceOf[JsString].value.equals("fail") &&
           fields.get("message").isDefined &&
-          fields.get("message").get.asInstanceOf[JsString].value.contains("failed to process inputs")
+          fields.get("message").get.asInstanceOf[JsString].value.contains("Workflow input processing failed.") &&
```

Comment:
Hey! I just re-added this like 2 months ago! Do we have an infiltrator with a campaign of && removal!?